test/e2e/manifest.yaml

```diff
@@ -14,3 +14,12 @@ DriverInfo:
   SupportedSizeRange:
     Min: 1Gi
     Max: 16Ti
+  Capabilities:
+    persistence: true
+    # our CI uses CephFS / NFV driver, which doesn't support fsgroups.
+    fsGroup: false
+    exec: true
+    # our CI uses CephFS / NFV driver, which doesn't support snapshots.
```

Comment:
NFS*

clarifying detail: The driver supports snapshots in the wallaby release of OpenStack Manila with Ceph Octopus+;